00_hello/hello_spec.rb

```diff
@@ -130,3 +130,7 @@
     greet("Bob").should == "Hello, Bob!"
   end
 end
+
+def greet(who)
```

Comment:
The `_spec` files are used to store only the tests for our application, it doesn't make sense to define here a method that will be used in other places.